librz/util/file.c

```diff
@@ -1183,6 +1183,32 @@ RZ_API bool rz_file_copy(const char *src, const char *dst) {
 #endif
 }
 
+RZ_API bool rz_file_gzip(const char *src) {
+	/* TODO: implement gzip for Windows */
+#if __WINDOWS__
+	return true;
+#else
+	char *src2 = rz_str_replace(strdup(src), "'", "\'", 1);
+	int rc = rz_sys_cmdf("gzip -f '%s'", src2);
```

Comment:
The whole function is wrong. Why do you need to call `gzip` if you already have `zlib`? Just use these `inflate`/`deflate` functions.

Reply:
Yes. @thestr4ng3r informed me to use `rz_deflate` and `rz_inflate` functions. I'll be refactoring it now. I'll squash the commits in the end (to keep the commit history clean).